apps/gp2-server/test/fixtures/event.fixtures.ts

```diff
@@ -33,6 +282,30 @@ export const getEventSpeakerUser = (): gp2Model.EventSpeakerUser => ({
   avatarUrl: undefined,
 });
 
+export const getContentfulEventDataObject = (): gp2Model.EventDataObject => ({
+  ...getEventDataObject(),
+  ...eventUnreadyResponse,
+  notes: '<p>These are the notes from the meeting</p>',
+  presentation:
+    '<p><iframe src="https://drive.google.com/file/d/1euARcFkYJoqj4c_oor_nYIMBDWOfL-3x/preview"/></p>',
```

Comment:
not sure these links contain private information, should it be better to use something like http://example.com/?